src/App.js

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect } from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import SplashPage from './components/SplashPage';
+import RegistrationForm from './components/session/RegistrationForm';
+import LoginForm from './components/session/LoginForm';
+import HomePage from './components/HomePage';
+import AddItemForm from './components/AddItemForm';
+import AllReservations from './components/AllReservations';
+import { fetchAllReservations } from './redux/reservations/reservationsSlice';
+import DoctorDetails from './components/DoctorDetails';
+import AddDoctor from './components/AddDoctorForm';
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor this functional component to ES6 arrow function, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

That way, your code will meet the latest standards. 💪